packages/backend/src/post/types/GraphQLPost.ts

```diff
@@ -18,9 +18,9 @@
 
 import {GraphQLNonNull, GraphQLObjectType, GraphQLString} from 'graphql';
 import {connectionDefinitions} from 'graphql-relay';
-import ProfileGraphQLType from '@/profile/types/ProfileGraphQLType';
+import GraphQLProfile from '@/profile/types/GraphQLProfile';
 
-const PostGraphQLType = new GraphQLObjectType({
+const GraphQLPost = new GraphQLObjectType({
```

Comment:
Maybe naming as `PostType` brings to you a best dx on future

since it will be a Type prefix you will know that is about GraphQL Types. See an example of naming here: 
- https://github.com/daniloab/rbaf-graphql-api/tree/main/src/server/modules/user
- UserModel.ts
- UserType.ts
- UserLoader.ts